### DISCUSSION TITLE
Is import of PointInstancer Supported?

### DISCUSSION THREAD

**Discussion by zoctipus** [isaac-sim/IsaacLab#1098] [Q&A]
I wanted to import a Rope, as demostrated in Window->simulation->demo scene

I cleaned up the usd such that only the Rope (PointInstancer and jointInstancer) are left
the cleaned up file can be found here:
[https://drive.google.com/file/d/1USnzZFNHi1nnxEJfZspYwpWv3zfkT8j7/view?usp=drive_link](url)

however when such usd is imported into env
as
```
rope = AssetBaseCfg(
        prim_path="{ENV_REGEX_NS}/Rope",
        spawn=sim_utils.UsdFileCfg(
            usd_path="/PATH/TO/ROPE/rope.usd",
        )
    )
```

while the cloner will make a copy of pointinstancer to all envs, but all of then are inactive and invisible.

What is causing this and if there's any solution for it? Thank you

**Comment by Dhoeller19** [comment]
Hi @zoctipus, unfortunately, this is not possible at the moment.
